src/manager.ts

```diff
@@ -74,22 +95,25 @@ export class SecretsManager implements ISecretsManager {
    * is programmatically filled.
    */
   async attach(
+    token: symbol,
     namespace: string,
     id: string,
     input: HTMLInputElement,
     callback?: (value: string) => void
   ): Promise<void> {
-    const attachedId = Private.buildSecretId(namespace, id);
+    Private.checkNamespace(token, namespace);
+    const attachedId = Private.buildConnectorId(namespace, id);
     const attachedInput = this._attachedInputs.get(attachedId);
```

Comment:
`_attachedInputs` should be in the private namespace, because i could do this in my extension

```js

secretManager['_attachedInputs'].forEach(it => {
   it.addeventlistener(...)
})

```

and get the value of the input when the secret value is filled.

Reply:
Yes, I realized that as well, but your extension can also do 
```
document.querySelector('[data-namespace]')
```
And I don't think that we can do anything about that.

Reply:
You don't have to save namespace and secret id to the input, just reverse the _attachedInputs map is enough to find these value from input element

Reply:
Yes, I agree that it would still be better to not expose it, but the extension can still do
```
document.querySelectorAll('input[type=password]')
```